### PR TITLE
Sambit/roger 1101

### PR DESCRIPTION
**Pull request by 04msambit**
The PR addresses the following changes:

1 - All events will end up as single metrics "statsd_rogeros_tools_exec_time".
2 - The records will have an "event" tag to identify the event.
3 - A new column (response_code) added.
4 - A new column "task_id" added to push / deploy.
5 - "Outcome" column fixed. Deploy is considered "Pass" only if pull/build/push succeeds.
6 - Added a counter metric to log push events. [ This will have the SOURCE column which will currently have the values TOOLS.] The measurement is called "statsd_rogeros_events".
7 - Fixed / Cleaned some existing code.

Test:

Checked generation of metric by deploying an sample_app in local cluster.
Checked the values in "statsd_rogeros_events".

@amitbose327 Please have a look

### CODE REVIEW

File: cli/roger_push.py [hunk 1]
```diff
@@ -390,16 +407,56 @@ def main(self, settings, appConfig, frameworkObject, hooksObj, args):
...
-                            sc.timing(input_metric, time_take_milliseonds)
+                            input_metric = "roger-tools.rogeros_tools_exec_time," + "app_name=" + str(args.app_name) + ",event=push" + ",container_name=" + str(container_name) + ",identifier=" + str(self.identifier) + ",outcome=" + str(execution_result) + ",response_code=" + str(status_code) + ",config_name=" + str(config_name) + ",env=" + str(environment) + ",user=" + str(settingObj.getUser())
+                            tup = (input_metric, time_take_milliseonds)
+                            self.statsd_message_list.append(tup)
+
+                            if str(status_code).startswith("20"):
+                                metric = input_metric.replace("rogeros_tools_exec_time", "rogeros_events")
+                                metric = metric + ",source=TOOLS" + ",task_id=" + self.task_id
```
Comment (ankanm) [inline cli/roger_push.py]: minor - `tools` instead of `TOOLS`

Comment (04msambit) [inline cli/roger_push.py]: Changed TOOLS to tools

**Comment by ankanm** [comment]
LGTM

**Comment by amitbose327** [comment]
LGTM

**Pull request merged**